src/main/java/org/schemaspy/input/dbms/service/ColumnService.java

```diff
@@ -129,15 +129,15 @@ private void initColumnAutoUpdate(Table table, boolean forceQuotes) {
         // so we can ask if the columns are auto updated
         // Ugh!!!  Should have been in DatabaseMetaData instead!!!
         StringBuilder sql = new StringBuilder("select * from ");
-        sql.append(sqlService.getQualifiedTableName(
-                table.getCatalog(),
-                table.getSchema(),
-                table.getName(),
-                forceQuotes)
-        );
+        String tableName=sqlService.getQualifiedTableName(
```

Comment:
Store the qualified table name to use both in the sql and in the exception message